user/enterprise/high-availability.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Enterprise High Availability Mode
+layout: en_enterprise
+
+---
+
+Travis CI Enterprise typically runs as a single container instance communicating with one or multiple workers and generally, this setup is quite stable and maintainable. However, in some cases -- especially volume licensing or by corporate requirement -- you might need to install Travis CI in High Availability (HA) mode. 
+
+If you're interested, or might be interested, in HA, please let [us](mailto:enterprise@travis-cicom?subject:HA%20Mode) know and we can discuss options and help you get started. 
+
+## Overview of Installation
+
+The platform installation is quite similar to the standard [Enterprise installation](/user/enterprise/installation#Setting-up-the-Travis-CI-Enterprise-Platform), and the [worker installation is identical](#Installing-the-Worker-in-High-Availability-Mode). However, there are some additional [system prerequisites](/user/enterprise/prerequisites/#High-Availability-Mode), which mean that to install in HA mode, you will need the following:
```

Comment:
"quite" doesn't add much here

maybe take "is identical" out of the link brackets?